StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Vertical Main Screen/RightAccessoryButton.swift

```diff
@@ -53,7 +58,12 @@ extension RowButton {
         private var imageView: UIImageView? {
             guard let image = accessoryType.accessoryImage else { return nil }
             let imageView = UIImageView(image: image)
-            imageView.tintColor = appearance.colors.primary // TODO(porter) use secondary action color
+            if accessoryType == .update {
+                imageView.tintColor = appearance.colors.icon
+            }
+            else {
+                imageView.tintColor = appearance.colors.primary // TODO(porter) use secondary action color
+            }
```

Comment:
Let's move this out to be a property on `AccessoryType`  like `text` and `accessoryImage`

Reply:
Ah nvm, we don't have an appearance object in there, let's leave it.